__tests__/integration.test.js

```diff
@@ -41,6 +41,34 @@ describe("api/topics", () => {
     });
 });
 
+describe("/api/articles/:article_id", () => {
+    test("GET:200 Should send the requested object on a key of 'article'", () => {
+        return request(app)
+            .get("/api/articles/4")
+            .expect(200)
+            .then((response) => {
+                const article = response.body.article;
+                expect(Object.keys(article).length).toBe(8);
+                expect(typeof article.author).toBe("string");
+                expect(typeof article.title).toBe("string");
+                expect(typeof article.article_id).toBe("number");
+                expect(typeof article.body).toBe("string");
+                expect(typeof article.topic).toBe("string");
+                expect(typeof article.created_at).toBe("string");
+                expect(typeof article.votes).toBe("number");
+                expect(typeof article.article_img_url).toBe("string");
+            });
+    });
+    test("GET:404 If a valid id is given, but id does not exist in the database, an arror message should be sent", () => {
+        return request(app)
+            .get("/api/articles/9999")
+            .expect(404)
+            .then((response) => {
+                expect(response.body.msg).toBe("Article not found");
+            });
+    });
+});
```

Comment:
It looks like you might be a missing a 400 test for an invalid article id ie api/articles/banana